active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+
+    Windows.UI.Xaml.Controls.IconElement Icon{ get; set; };
+
+    [MUX_DEFAULT_VALUE("true")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsCloseable{ get; set; };
+
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IconProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsCloseableProperty{ get; };
+
+    // TODO: Add
+    event Windows.Foundation.TypedEventHandler<TabViewItem, TabViewTabClosingEventArgs> TabClosing;
+}
+
+}
+
+namespace MU_XCP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewListView : Windows.UI.Xaml.Controls.ListView
+{
+    TabViewListView();
+}
+
+}
+
+namespace MU_XAP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewAutomationPeer : Windows.UI.Xaml.Automation.Peers.FrameworkElementAutomationPeer
+{
+    TabViewAutomationPeer(MU_XC_NAMESPACE.TabView owner);
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewItemAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewItemAutomationPeer
+{
+    TabViewItemAutomationPeer(MU_XC_NAMESPACE.TabViewItem owner);
+}
+
+}
+```
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+![Position of TabStripHeader and TabStripFooter](images/tab-control-parts-1.PNG)
+
+![Parts of a tab item](images/tab-control-parts-2.PNG)
+
+## Detailed Keyboarding Behavior
+
+This section discusses the built-in keyboarding functionality of the TabView control. The [Keyboard Guidance for App Developers](#Keyboard-guidance-for-App-Developers) section discusses specific recommendations to extend the keyboarding functionality for apps using TabView.
+
+### TAB and Arrow key behavior in the TabStrip
+When focus moves into the TabStrip area, the selected TabViewItem gains focus. The user can then use the LEFT/RIGHT arrow keys to move focus (not selection) to other tabs in the TabStrip. Arrow focus is trapped inside the Tab content area. To move focus out of the TabStrip area, the user can press the TAB key which will move focus to the next focusable element. 
+
+_Move focus via TAB_
+
+![Move focus via TAB](images/tab-control-keyboarding-1.PNG)
+
+_ARROW does not cycle focus_
+
+![ARROW does not cycle focus](images/tab-control-keyboarding-3.PNG)
+
+### Selecting a tab
+When a TabViewItem has focus, pressing SPACE or ENTER will select that TabViewItem.
+
+_Use ARROW to move focus, ENTER to select tab_
+
+![Use ARROW to move focus, ENTER to select tab](images/tab-control-keyboarding-2.PNG)
+
+### Shortcuts for selecting adjacent tabs
+Ctrl+TAB will select the next TabViewItem.
+Ctrl+Shift+TAB will select the previous TabViewItem.
```

Comment:
When are the <kbd>Ctrl</kbd>+<kbd>TAB</kbd> and <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>TAB</kbd> shortcuts active?

* When focus is on the TabStrip.
* When focus is anywhere in the TabView.
* When focus is anywhere in the window. (What if there is more than one TabView in the window?)

If the app has controls outside the TabView (e.g., status bar at bottom), and it wants <kbd>Ctrl</kbd>+<kbd>TAB</kbd> to change tabs, then it presumably needs to install its own global shortcut. Will that interact adversely with the built-in shortcut?

Reply:
Discussion: When focus is anywhere inside the tab view (including the tab strip).